File: back/src/main/java/com/back/domain/book/repository/BookFetchProgressRepository.java
package com.back.domain.book.repository;

import com.back.domain.book.entity.BookFetchProgress;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface BookFetchProgressRepository extends JpaRepository<BookFetchProgress, Long> {
    Optional<BookFetchProgress> findFirstByOrderByIdAsc();
}

File: back/src/main/kotlin/com/back/domain/book/repository/BookFetchProgressRepository.kt
package com.back.domain.book.repository

import com.back.domain.book.entity.BookFetchProgress
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface BookFetchProgressRepository : JpaRepository<BookFetchProgress, Long> {
    fun findFirstByOrderByIdAsc(): BookFetchProgress?
}